packages/dx-chart-core/src/plugins/series/computeds.test.js

```diff
@@ -481,7 +479,43 @@ describe('addSeries', () => {
           },
         ],
         color: 'c2',
-        palette,
+      },
+    ]);
+  });
+
+  it('should create points for pie', () => {
```

Comment:
There is no *pie* specific here - it should be something like *'should calculate point color'* or *'should calculate point color with "getPointColor" function'*.